pkg/detector/detector.go

```diff
@@ -206,6 +191,19 @@ func (d *ResourceDetector) Reconcile(key util.QueueKey) error {
 	}
 	klog.Infof("Reconciling object: %s", clusterWideKey)
```

Comment:
Can we move the log down? I don't want the ignored resources present in the logs by default.